images/linux/scripts/installers/swift.sh

```diff
@@ -15,6 +15,17 @@ swift_tar_name="swift-$swift_version-RELEASE-ubuntu$image_label.tar.gz"
 swift_tar_url="https://swift.org/builds/swift-$swift_version-release/ubuntu${image_label//./}/swift-$swift_version-RELEASE/$swift_tar_name"
 download_with_retries $swift_tar_url "/tmp" "$swift_tar_name"
 
+# Verifing pgp signature using official swift pgp key. Referring to https://www.swift.org/install/linux/#Installation-via-Tarball
+# Download swift pgp key
+download_with_retries "https://swift.org/keys/all-keys.asc" "/tmp" "all-keys.asc"
+# Import swift pgp key
+gpg --import /tmp/all-keys.asc
```

Comment:
Are we good with keeping this public key in keyring? If not then we should either remove it afterwards or import it to temporary keyring instead (using `--no-default-keyring --keyring <keyring-file-name>`)

Reply:
I didn`t see nothing bad in keeping keys. @mikhailkoliada What do you think?

Reply:
yes, lets clean it up after we use it

Reply:
fixed